### PR TITLE
Alumni Sans: Version 1.018 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
0188f4b: [gftools-packager] Alumni Sans: Version 1.018 added

* Alumni Sans Version 1.018 taken from the upstream repo https://github.com/googlefonts/alumni at commit https://github.com/googlefonts/alumni/commit/28754a9295db605d4e4ffc7bf60b4a8301eef9ab.

 edc9932: [gftools-packager] ofl/alumnisans remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Updated

## Alumni Sans: Version 1.018 added

---
 61cee8e: [gftools-packager] Alumni Sans: Version 1.018 added

* Alumni Sans Version 1.018 taken from the upstream repo https://github.com/googlefonts/alumni at commit https://github.com/googlefonts/alumni/commit/44a7998fa2bfa1b3e119983cdc565dd7f0f03bda.

 65d5f2a: [gftools-packager] ofl/alumnisans remove METADATA "source".  google/fonts#2587

**Comment by vv-monsalve** [comment]
Updated

## Alumni Sans: Version 1.018 added

---
 a787253: [gftools-packager] Alumni Sans: Version 1.018 added

* Alumni Sans Version 1.018 taken from the upstream repo https://github.com/googlefonts/alumni at commit https://github.com/googlefonts/alumni/commit/44a7998fa2bfa1b3e119983cdc565dd7f0f03bda.

 e10acb8: [gftools-packager] ofl/alumnisans remove METADATA "source".  google/fonts#2587

File: ofl/alumnisans/METADATA.pb [hunk 1]
```diff
@@ -32,221 +32,3 @@ axes {
...
-languages: "war_Latn"  # Waray
-languages: "wbp_Latn"  # Warlpiri
-languages: "wo_Latn"  # Wolof
-languages: "xav_Latn"  # Xavánte
-languages: "xh_Latn"  # Xhosa
-languages: "xog_Latn"  # Soga
-languages: "yao_Latn"  # Yao
-languages: "yap_Latn"  # Yapese
-languages: "yua_Latn"  # Yucateco
-languages: "zu_Latn"  # Zulu
```
Comment (RosaWagner) [inline ofl/alumnisans/METADATA.pb:252]: @m4rc1e I saw the same issue in another PR, what do we do until fixed? We re-add the languages manually?

Comment (m4rc1e) [reply ofl/alumnisans/METADATA.pb:252]: Yes, please readd them. I have an issue open https://github.com/googlefonts/gftools/issues/456. Personally, I wouldn't PR any family upgrade until this is fixed.

**Comment by vv-monsalve** [comment]
Thanks @RosaWagner for re-adding the lang support.

**Comment by vv-monsalve** [comment]
@RosaWagner, as the METADATA.pb includes now the lang support list, I think it es ready for review.

**Pull request merged**